### PR TITLE
Transit Fares in Directions & DistanceMatrix

### PR DESCRIPTION
**Pull request by markmcd**
Part of #59. PTAL @domesticmouse @broady 

There's more to come.

### CODE REVIEW

**Comment by broady** [comment]
excellent

**Comment by markmcd** [comment]
[LGTM](http://www.lgtm.in/i/ReE1)

**Comment by markmcd** [comment]
OK I'm finished now :)

**Comment by markmcd** [comment]
Any last comments here @broady? Or good to merge?

**Comment by broady** [comment]
Just the one. otherwise LGTM

File: src/main/java/com/google/maps/DirectionsApiRequest.java [hunk 1]
```diff
@@ -184,4 +185,21 @@ public DirectionsApiRequest alternatives(boolean alternateRoutes) {
       return param("alternatives", "false");
     }
   }
+
+  /**
+   * Specifies one or more preferred modes of transit. This parameter may only be specified for
+   * requests where the mode is transit.
+   */
+  public DirectionsApiRequest transitModes(TransitMode... transitModes) {
```
Comment (broady) [inline src/main/java/com/google/maps/DirectionsApiRequest.java]: `transitMode`

**Pull request merged**